form/src/main/kotlin/com/ritense/form/service/FormSubmissionService.kt

```diff
@@ -77,26 +78,29 @@ open class FormSubmissionService(
         taskInstanceId: String?,
     ): FormSubmissionResult {
         return try {
-            val processLink = processLinkService.getProcessLink(processLinkId, FormProcessLink::class.java)
-            val document = documentId?.let { documentService.get(documentId) }
-            val processDocumentDefinition = getProcessDocumentDefinition(processLink, document)
-            val processVariables = getProcessVariables(taskInstanceId)
-            val formDefinition = formDefinitionService.getFormDefinitionById(processLink.formDefinitionId).orElseThrow()
-            val formFields = getFormFields(formDefinition, formData)
-            val submittedDocumentContent = getSubmittedDocumentContent(formFields, document)
-            val formDefinedProcessVariables = formDefinition.extractProcessVars(formData)
-            val preJsonPatch = getPreJsonPatch(formDefinition, submittedDocumentContent, processVariables, document)
-            val externalFormData = getExternalFormData(formDefinition, formData)
-            val request = getRequest(
-                processLink,
-                document,
-                taskInstanceId,
-                processDocumentDefinition,
-                submittedDocumentContent,
-                formDefinedProcessVariables,
-                preJsonPatch
-            )
-            return dispatchRequest(request, processDocumentDefinition, formFields, externalFormData)
+            return AuthorizationContext.runWithoutAuthorization {
```

Comment:
Why is authorization disabled here? not everyone is allowed to submit a form

Reply:
Endpoint itself is still protected, just like it was before. Support for forms will not be included in major 11.